resources/pdns_authoritative_service_sysvinit.rb

```diff
@@ -0,0 +1,92 @@
+#
+# Cookbook Name:: pdns
+# Resources:: pdns_authoritative_service
+#
+# Copyright 2017, Aetrion, LLC DBA DNSimple
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+include ::PdnsAuthoritativeResource::Helpers
+
+resource_name :pdns_authoritative_service_sysvinit
+
+provides :pdns_authoritative_service, os: 'linux' do |_node|
+  Chef::Platform::ServiceHelpers.service_resource_providers.include?(:debian) ||
+    Chef::Platform::ServiceHelpers.service_resource_providers.include?(:redhat)
+end
+
+property :instance_name, String, name_property: true
+property :cookbook, String, default: 'pdns'
+property :source, %w(String NilClass)
```

Comment:
That would the case if the templating is the default use which not, the link is, woudn't it?